reference/configuration/framework.rst

```diff
@@ -1497,14 +1497,16 @@ If this option is enabled, serialization groups can be defined using annotations
 
     For more information, see :ref:`serializer-using-serialization-groups-annotations`.
 
+.. _reference-serializer-name_converter:
+
 name_converter
 ..............
 
+**type**: ``string``
+
 .. versionadded:: 2.8
     The ``name_converter`` setting was introduced in Symfony 2.8.
```

Comment:
`The xxx setting was introduced ...`  is used 3 times in Symfony Docs, but `The xxx option was introduced...` alternative is used tens of times. Let's normalize that in a separate PR.